.github/workflows/ci_pullrequest.yml

```diff
@@ -275,9 +317,7 @@ jobs:
       - name: "Run coverage"
         run: "poetry run invoke unittest-coverage"
     needs:
-      - "pylint"
-      - "tests-postgres"
-      - "tests-mysql"
+      - "performance-test"
```

Comment:
I'd prefer to run integration and performance tests in parallel so that we're not adding another serial step to our already lengthy CI process.